app.js

```diff
@@ -51,8 +52,22 @@ app.use(function(req, resp, next){
   }
 });
 
+// strings, and navs
 app.use(reactRouted);
 app.use(express.static(__dirname + '/public', {maxAge: 3600000}));
+app.use(function(req, res, next) {
```

Comment:
This handles any 404's that is, anything we don't find a match for in reactRouted or static files, goes in here.

While in here, we do a locale redirect if applicable, otherwise we do a next call.